web_src/css/base.css

```diff
@@ -2238,10 +2238,13 @@ a.ui.active.label:hover {
   border-left: none;
 }
 
+/* a ghost button can be used as inline text, it doesn't have obvious styles */
 .ui.button.button-ghost {
```

Comment:
The only thing I'm not yet sure of is if we can drop the `.ui` class for this selector.
I don't think it is necessary, and there shouldn't be any false-positive `button button-ghost` buttons yet as it is still absolutely new…
And I don't think we can expect new contributors to know that they need the `ui` class as well.

Reply:
Agree to the `ui` removal. We should not make it look like this is an fomantic override. It's actually an extension to it.

Reply:
Re-use the `ui button` can re-use other styles like colors, `mini/tiny`, etc.

Reply:
So, looks like two people in favor of removing the `ui` prefix to me.

Reply:
`mini` etc will still work. BTW, I think we should move button styles to a separate file, I will do that in https://github.com/go-gitea/gitea/pull/24507 now, so if this merges first, I will include moving this.

Reply:
removed it.